### DISCUSSION TITLE
Chapters not always showing on media-time-range element

### DISCUSSION THREAD

**Discussion by endymion1818** [muxinc/media-chrome#916] [Q&A] [answered]
Hi

I'm having issues with media-chrome and `mux-video`, I have a chapters.vtt which I'm adding to the `mux-video` element, and the chapters are showing in the track, however chapter titles aren't showing when you hover across the timeline.

When clicking on the 1px / 2px area of the exact chapter:
<img width="672" alt="Screenshot 2024-05-24 at 14 44 30" src="https://github.com/muxinc/media-chrome/assets/2216344/71f40b49-b148-433b-8273-0ee44192977d">

When clicking elsewhere on the timeline:
<img width="674" alt="Screenshot 2024-05-24 at 14 44 43" src="https://github.com/muxinc/media-chrome/assets/2216344/81dd025d-e9d1-46f3-9efa-0685225e8562">

Versus demo on https://www.media-chrome.org/docs/en/components/media-time-range:
<img width="456" alt="Screenshot 2024-05-24 at 14 53 05" src="https://github.com/muxinc/media-chrome/assets/2216344/7c7d6427-0971-41ae-bab2-c47cda92187f">

I'm mentioning it since I can't identify what might be causing this issue. The VTT file seems to be in the same format as the ones on your demos.

Minimum reproducible example: https://codepen.io/endymion1818/pen/VwOKvKK

**Comment by endymion1818** [comment]
Can I also say that this is a minor niggle, we're in the process of replatforming and these components are an absolute joy to work with.

**Comment by luwes** [comment]
thanks for reporting! this looks like a bug, we will have a look at this soon.

**Comment by luwes** [comment, marked as answer]
actually it's not a bug. take a look at the length of the chapters, it's just 1s each time. 

chapters need to indicate the start and end times correctly. you probably want to line up the end times with the start times of the next chapter to display this correctly.

**Reply by endymion1818** [reply]
Oh didn't spot that. Thanks.
